src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService serv = new UserService();
+        User user = serv.findByEmail(email);
+        if (user != null) {
+            if (user.getPassword().equals(password)) {
+                return true;
+            }
+        }
+        System.out.println("No user with this e-mail is found");
         return false;
```

Comment:
try to rewrite this solution without using boolean literals (true/false) at all. Use logical expressions directly in return